### ISSUE TITLE
Using the `each` method with Scala 2.10

### ISSUE THREAD

**Issue by ncreep** [adamw/quicklens#10]
Hi,

I'm trying to run Quicklens `1.4.0` with Scala `2.10.6` and the `each` method fails to compile. For example this code:
```scala
import com.softwaremill.quicklens._

object Test extends App {
  case class Foo(bar: Option[Int])

  val foo = Foo(Some(5))
  foo.modify(_.bar.each).setTo(16)
}

```
fails to compile with the error:
```
1 not in List(value $outer, value 1)
```
and an output with the generated AST.

While the code
```scala
import com.softwaremill.quicklens._

object Test extends App {
  case class Foo(bar: Option[Int])

  val foo = Foo(Some(5))
  foo.modify(_.bar).setTo(Some(16))}

```
works as expected.

Is it possible to use the `each` method on 2.10?

Thanks.

**Comment by stanch** [comment]
Apparently it’s caused by this: https://issues.scala-lang.org/browse/SI-8533.
I just did a quick replace of `c.fresh()` → `c.fresh() + "0"` and all the tests passed.
Gonna cherry-pick my newest features on top and send a PR soon.

**Comment by adamw** [comment]
Released as 1.4.4_2.10 - thanks! :)
